src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,14 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+export const categoriesSlice = createSlice({
+  name: "category",
+  initialState: [],
+  reducers: {
+    addCategory: (state, action) => {
+      state.push(action.payload);
+    },
```

Comment:
Nice job so far😉. Kindly note that the requirement of the project states that your `categoriesSlice` Includes a reducer that checks the status and always returns "Under construction"😉.currently, you have implemented an `addCategory reducer which is not required🥲. Let's try to stick to the requirement of the project👋👍.
<img width="743" alt="image" src="https://github.com/DilsherB/bookstore/assets/64643995/ad5390cb-6e84-465d-b17a-e3d25350e26a">

```suggestion
  reducers: {
    checkStatus: (state, action) => {
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories
    },
  },

```

Reply:
@Ibizugbe Thank you for your review and nice explanation of the changes required.